gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -467,14 +496,15 @@ gst_tensor_query_client_sink_event (GstPad * pad,
     case GST_EVENT_CAPS:
     {
       GstCaps *caps;
-      gchar *_topic = NULL, *protocol_str = NULL;
+      gchar *_topic = NULL;
 
       if (self->topic)
         _topic = g_strdup (self->topic);
       else
         _topic = g_strdup ("TCP_DIRECT");
 
-      nns_edge_create_handle ("TEMP_ID", _topic, &self->edge_h);
+      nns_edge_create_handle ("TEMP_ID", self->connect_type,
+          NNS_EDGE_FLAG_ALL, &self->edge_h);
```

Comment:
recommend not to use NNS_EDGE_FLAG_ALL (this may include unnecessary flags later. set `NNS_EDGE_FLAG_RECV | NNS_EDGE_FLAG_SEND` instead.)